ChangeLog

```diff
@@ -1,4 +1,6 @@
 3.18.6:
+	- Fixed and improved postgresql server state handling in package scriptlets
```

Comment:
maybe "package" should be changed to "installer"? @nickanderson @olehermanse @larsewi

Reply:
How about package installer scriptlets?